ivy/functional/frontends/tensorflow/math.py

```diff
@@ -453,6 +453,15 @@ def round(x, name=None):
     return ivy.round(x)
 
 
+@with_supported_dtypes(
+    {"2.9.0 and below": ("float16", "float32", "float64", "complex64", "complex128")},
+    "tensorflow",
```

Comment:
Can you please add `bfloat16` here. I think this should resolve the error in `intelligent-tests/run_test_1`